staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/resources.go

```diff
@@ -258,35 +258,41 @@ func CreateNewCustomResourceDefinition(crd *apiextensionsv1beta1.CustomResourceD
 	// For this test, we'll actually cycle, "list/watch/create/delete" until we get an RV from list that observes the create and not an error.
 	// This way all the tests that are checking for watches don't have to worry about RV too old problems because crazy things *could* happen
 	// before like the created RV could be too old to watch.
-	for _, version := range servedVersions(crd) {
-		err := wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
-			return isWatchCachePrimed(crd, dynamicClientSet, version)
-		})
-		if err != nil {
-			return nil, err
-		}
+	err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
+		return isWatchCachePrimed(crd, dynamicClientSet)
```

Comment:
nit: no need for the func, pass isWatchCachePrimed directly (eta conversion...).

Reply:
How do I pass crd  and dynamicClientSet to it?

Reply:
nvmd. Misread.